test/model/course/course_test.dart

```diff
@@ -228,5 +266,42 @@ void main() {
       expect(CourseAvailabilityUtility.getFacultyAsInt(CourseAvailability.UNAVAILABLE), 2);
     });
   });
+
+  group('weekday utility', () {
+    test('1', () {
+      expect(WeekdayUtility.getWeekdayAsString(Weekday.Sun), "Sun");
+    });
+
+    test('2', () {
+      expect(WeekdayUtility.getWeekdayAsString(Weekday.Sat), "Sat");
+    });
+    test('3', () {
+      expect(WeekdayUtility.getWeekdayAsString(Weekday.Fri), "Fri");
+    });
+
+    test('4', () {
+      expect(WeekdayUtility.getWeekdayAsInt(Weekday.Sun), 6);
+    });
+
+    test('5', () {
+      expect(WeekdayUtility.getWeekdayAsInt(Weekday.Sat), 5);
+    });
+    test('6', () {
+      expect(WeekdayUtility.getWeekdayAsInt(Weekday.Fri), 4);
+    });
+
+    test('7', () {
+      expect(WeekdayUtility.intToWeekday(4), Weekday.Fri);
+    });
+
+    test('8', () {
+      expect(WeekdayUtility.intToWeekday(5), Weekday.Sat);
+    });
+    test('9', () {
+      expect(WeekdayUtility.intToWeekday(6), Weekday.Sun);
+    });
+
+    //could do more.... but others are already covered
```

Comment:
I think it would be a good approach to test all of them, should not be to much work, I guess.